packages/html/src/grid/tests/grid-adaptive-pager.tsx

```diff
@@ -17,24 +17,7 @@ root.render(
                         <div className="k-view k-widget">
                             <div className="k-stretched-view k-content">
                                 <div className="k-grid k-widget k-grid-display-block k-grid-mobile k-editable">
-                                    <div className="k-pager k-grid-pager k-widget k-pager-sm">
-                                        <a className="k-link k-pager-nav k-pager-first"><Icon name="arrow-end-left" /></a>
-                                        <a className="k-link k-pager-nav"><Icon name="arrow-w" /></a>
-                                        <div className="k-pager-numbers-wrap">
-                                            <select className="k-picker k-dropdown-list k-dropdown k-rounded-md">
-                                                <option>1</option>
-                                            </select>
-                                            <ul className="k-pager-numbers">
-                                                <li><a className="k-link">1</a></li>
-                                                <li><a className="k-link">2</a></li>
-                                                <li><a className="k-link">3</a></li>
-                                                <li><span className="k-link k-selected">4</span></li>
-                                            </ul>
-                                        </div>
-                                        <a className="k-link k-pager-nav k-disabled"><Icon name="arrow-e" /></a>
-                                        <a className="k-link k-pager-nav k-pager-last k-disabled"><Icon name="arrow-end-right" /></a>
-                                        <span className="k-pager-info k-label">61 - 77 of 77 items</span>
-                                    </div>
+                                    <Pager size="small" className="k-grid-pager" mobile="small" showPagerInput={false} />
```

Comment:
I think we should keep size="medium" (or omit the size) here, so that the images remain unchanged.